src/react/databrowser/objects/ObjectRow.jsx

```diff
@@ -35,21 +42,38 @@ type RowProps = {
 export const createItemData = memoize((
     rows: RowsType,
     prepareRow: PrepareRow,
+    objectName: ?string,
+    dispatch: DispatchAPI<Action>,
+    bucketName: string,
+    prefixWithSlash: string,
 ): Data => ({
     rows,
     prepareRow,
+    objectName,
+    dispatch,
+    bucketName,
+    prefixWithSlash,
 }), isDeepEqual);
 
 // https://react-window.now.sh/#/examples/list/memoized-list-items
 const Row = ({
-    data: { rows, prepareRow },
+    data: { rows, prepareRow, objectName, dispatch, bucketName, prefixWithSlash },
     index,
     style,
 }: RowProps) => {
     const row = rows[index];
     prepareRow(row);
+    const isSelected = objectName && objectName === row.original.name;
+
+    const handleClick = () => {
+        if (row.original.isFolder || isSelected) {
+            return;
```

Comment:
The way we designed it was:
- user can check (using checkboxes) multiple rows (objects or folders) and apply an action on then (ie deletion). I use the verb "toggle" for that kind of "selection" in the code. 

- user can select only one row (object only) at a time by clicking inside the Row. It will show the object metadata. I use the verb "select" for that in the code.

- User clicks on the Row name to either get into a folder or download an object.

Reply:
@terrydervaux @dvasilas We need to rethink the design since it is confusing to have the checkbox and the "row selection" that are not synchronized.

Reply:
I've had this feedback several times on orbit's browser (which is s3browser), because people are already used to selecting objects (emails in gmail, files in the Finder, aws s3's object browser, etc) and expect a certain behavior, which is generally different from the "highlighting a row is different than checking checkbox(es)" s3browser behavior with regards to what displays in the sidebar/statusbar and what is affected by actions on the selection.

Reply:
- The checkbox and the "row selection" are now synchronized. 
- If a folder is selected, the MD pane shows: "Select an object."
- If multiple items (objects, folders or both) are selected, the MD pane shows: "Multiple items selected."
I updated the video on the description.